legal-api/src/legal_api/services/filings/validations/amalgamation_application.py

```diff
@@ -86,7 +86,15 @@ def validate_amalgamating_businesses(  # pylint: disable=too-many-branches,too-m
     amalgamating_businesses = {}
     for amalgamating_business_json in amalgamating_businesses_json:
         if identifier := amalgamating_business_json.get("identifier"):
-            if not (business := BusinessService.fetch_business(identifier)):
+            if (
+                identifier.startswith("A")
+                and (foreign_jurisdiction := amalgamating_business_json.get("foreignJurisdiction"))
+                and foreign_jurisdiction.get("country") == "CA"
+                and foreign_jurisdiction.get("region") == "BC"
+            ):
+                is_any_expro_a = True
```

Comment:
We no longer use `corpNumber`, so this section of code had to moved out of the `elif` statement below.